packages/gatsby-theme/src/components/InfiniteScroll/index.js

```diff
@@ -40,10 +40,10 @@ InfiniteScroll.propTypes = {
 
 InfiniteScroll.defaultProps = {
   hasMore: false,
-  loadPlaceholder: null,
+  loadPlaceholder: undefined,
   scrollThreshold: "800px",
-  onScrollHandler: () => null,
-  endMessage: null,
+  onScrollHandler: () => undefined,
```

Comment:
What do you think that if we make the function undefined instead of return undefined?
```suggestion
  onScrollHandler: undefined,
```

Reply:
Sure! thanks @luisduenas 🙏